deploy-board/deploy_board/webapp/util_views.py

```diff
@@ -39,11 +39,17 @@ def _convert_opentsdb_data(dps):
 
 def _get_latest_metrics(url):
     response = urllib2.urlopen(url)
-    data = json.loads(response.read())
+    decoded_response = str(response.read()).decode('string_escape')
```

Comment:
Can you explain this a little more? This only works python2, we should try to avoid tech debt.

Reply:
Without this line it is unable to parse json object. I saw on removing escape sequences, it works. I can look for a python 3 alternative. Although even urllib2 is a python 2 package.

Reply:
`data = json.load(response)` worked for me.

Reply:
thanks, apologies for doing it in a convoluted manner